TESTS/mbed_hal_fpga_ci_test_shield/basic/main.cpp

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2013-2017, ARM Limited, All Rights Reserved
```

Comment:
```suggestion
 * Copyright (c) 2013-2019, ARM Limited, All Rights Reserved
```
Update the date if modified. This also applies for the following files.